vmdb/lib/miq_automation_engine/engine/miq_ae_method.rb

```diff
@@ -133,6 +133,8 @@ class AutomateMethodException < StandardError
   require 'drb'
   require 'yaml'
 
+  Time.zone = 'UTC'
```

Comment:
@gmcculloug @Fryguy you guys ok with setting this in the middle of a run?

Feeling like we should be setting this at the beginning of the automate job